pkg/rest/coverage/swagger_analysis.go

```diff
@@ -0,0 +1,119 @@
+package coverage
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-openapi/loads"
```

Comment:
I think this might be a bit cryptic. When used it only says `loads.JSONSpec`. In my opinion it would be better to import the library something like
```
import  apiload "github.com/go-openapi/loads"
```
It would than read much clearer.

Reply:
I prefer to keep as it is, from the pkg context it's easy to predict what it does.